src/constants/enums/geospatialCountries.ts

```diff
@@ -0,0 +1,6 @@
+export enum GeospatialCountriesEnum {
+  E = 'England',
```

Comment:
A minor point, perhaps list them out in alphabetic order please.

Reply:
Old order was based on personal preference, changed to alphabetical order.